src/apis/yelpActions.ts

```diff
@@ -1,92 +1,74 @@
-  "Rusty Taco": "fq9jC-Jl55VySyZ66-nN1A",
-  "Yeah Yeah Taco": "R5371sng0zz7icCHxlpsoQ",
-  "Nico's Taco and Tequila Bar": "HL8GOJJsauMCD8TET7UmNg",
-  "Maya Cuisine": "mdXoIXksYTiJhsYo-uknkQ",
-  "Galactic Pizza": "dBRIlzRYdvAhSdxOTBVYyw",
-  "Young Joni": "5Y2nekkwRrNMMSx3b-2EhA",
-  "Rinata": "9EdftCquw0YwWY3XsJlZVg",
-  "Lands End Pasty Company": "kLIRGu0p-j8ilVO-SfkWZw",
-  "Isles Bun & Coffee": "sdeYah2re9mvnHEsbIPsOw",
-  "Eggy's Diner": "cPgZaWUgM1pj3po6UAgtRw",
-  "Hen House Eatery": "ykTnu3sVxnnIeHpLRqE8bQ",
-  "Mi Tea": "43p95RkcHPHbCoJLFSpdeA",
-  "Ding Tea": "F38zcGJ2FerUQo--3dw1cA",
-  "Tiger Sugar": "U0nC_UNCVZoYKyJLWtTiWg"
-};
-
-export async function getLocalCoffeeShops(coordinates: Coordinates, currentAddress: string): Promise<Coffeeshop[]> {
+export async function getLocalCoffeeShops(
+  coordinates: Coordinates,
+  currentAddress: string
+): Promise<Coffeeshop[]> {
   try {
-    const res = await axios
-      .get(
-        `https://api.yelp.com/v3/businesses/search?term=coffee&latitude=${coordinates.lat}&longitude=${coordinates.long}&limit=48`,
-        {
-          headers: {
-            Authorization:  YELP_BEARER_TOKEN,
-          }
-        }
-      )
-      const coffeeshops = res.data.businesses.map((shop: any) => ({name: shop.name, address: shop.location.address1.concat(", ", shop.location.city, ", ", shop.location.state, " ", shop.location.zip_code), imageUrl: shop.image_url, drivingTime: 0}))
-
-      let promises: any[] = []
-      coffeeshops.map(async (shop: any) => 
-        promises.push(getDrivingTime(currentAddress, shop.address).then((result) => {return {...shop, drivingTime: result}})
-        )
+    const res = await axios.get(
+      `https://api.yelp.com/v3/businesses/search?term=coffee&latitude=${coordinates.lat}&longitude=${coordinates.long}&limit=48`,
+      {
+        headers: {
+          Authorization: YELP_BEARER_TOKEN,
+        },
+      }
+    )
+    const coffeeshops = res.data.businesses.map((shop: any) => ({
+      name: shop.name,
+      address: shop.location.address1.concat(
+        ", ",
+        shop.location.city,
+        ", ",
+        shop.location.state,
+        " ",
+        shop.location.zip_code
+      ),
+      imageUrl: shop.image_url,
+      drivingTime: 0,
+    }))
 
+    let promises: any[] = []
+    coffeeshops.map(async (shop: any) =>
+      promises.push(
+        getDrivingTime(currentAddress, shop.address).then((result) => {
+          return { ...shop, drivingTime: result }
+        })
       )
-      const coffeeshopsWithDrivingTime = await Promise.all(promises)
-      return coffeeshopsWithDrivingTime
+    )
+    const coffeeshopsWithDrivingTime = await Promise.all(promises)
+    return coffeeshopsWithDrivingTime
   } catch (err) {
-    throw(err)
+    throw err
   }
-};
+}
 
-export async function getBusinessDetails(name: string): Promise<Restaurant> {
+export async function getBusinessDetails(id: string): Promise<Restaurant> {
```

Comment:
this and line 50 are the only changes in this file, i guess i have prettier set up